pkg/harvester/models/kubevirt.io.virtualmachine.js

```diff
@@ -691,6 +691,17 @@ export default class VirtVm extends HarvesterResource {
   }
 
   get restoreProgress() {
+    const inStore = this.productInStore;
+    const allBackups = this.$rootGetters[`${ inStore }/all`](HCI.BACKUP);
+
+    const isSnapshotRestore = !!allBackups
+      .filter(b => b.spec?.type !== 'backup')
```

Comment:
How about moving this type (`backup`) to another file as const variable for easy management? We could move it step by step, no need to move all once.

Reply:
That's fine, code refactoring is good :+1: